react-js/react.md

```diff
@@ -41,29 +42,108 @@ You will spend the day working through these materials.
 
 ### Topics to learn & Facilitate
 
-- What is React?
-    - React is a library for creating User Interfaces.
-    - This means it is used for creating what some refer to as the "view layer" of an application.
-- JSX adds to our JavaScript to allow for HTML-like markup in JavaScript.
-- Show Hello World example, plus expanded example.
-- Example for appending to a list in jQuery vs. the same in React.
+- React lets you generate HTML using JavaScript
+- The HTML can change over time, unlike regular HTML
+- No jQuery needed
+
+### Examples
+- Hello World component
+```
+helloWorld.js
+
+import ReactDOM from 'react-dom'
```

Comment:
import isn't going to work out of the box. Can you establish these first in an online REPL, like CodePen or something that already has React set up? And then give a link for the reader so they can fork their own

Reply:
Done. I have added three codepen examples with their links in my latest commits.